GitAutomation/GitAutomation.psd1

```diff
@@ -16,7 +16,7 @@
     RootModule = 'GitAutomation.psm1'
 
     # Version number of this module.
-    ModuleVersion = '0.12.3'
+    ModuleVersion = '0.12.5'

```

Comment:
Reminder that this version needs to be `0.13.0` before we merge.

Reply:
Ok done.